### ISSUE TITLE
Update demo page to use v0.9.0.

### ISSUE THREAD

**Issue by kevinbarabash** [Khan/KaTeX#1171]
This change will need to be made on the `gh-pages` branch.

**Comment by ronkok** [comment]
@kevinbarabash What does this involve? I can see that the contents of `bower_components/katex/dist/` must be refreshed with files from the most recent release.

Is there anything else?

**Comment by kevinbarabash** [comment]
@ronkok that should be it.

**Comment by kevinbarabash** [comment]
It would be cool if we could automate this as part of the release script at some point.

**Comment by edemaine** [comment]
I tried running `bower update katex` on the `gh-pages` branch, and I got the following output:

```
bower not-cached    https://github.com/Khan/KaTeX.git#0.9.0-alpha2
bower resolve       https://github.com/Khan/KaTeX.git#0.9.0-alpha2
bower download      https://github.com/Khan/KaTeX/archive/v0.9.0-alpha2.tar.gz
bower extract       katex#0.9.0-alpha2 archive.tar.gz
bower resolved      https://github.com/Khan/KaTeX.git#0.9.0-alpha2
```

I conclude that bower katex somehow isn't updating... Any thoughts on how to fix this?

**Comment by ronkok** [comment]
Just out of curiosity, why does the demo page load its `katex.js` from the `bower_components` subdirectory? Why not get it from the cdnjs.cloudflare.com CDN like everyone else?

**Comment by edemaine** [comment]
@ronkok Nice idea.  My guess is that the demo site just predates CDN, but that seems like a good option too. One possible disadvantage is that the CDN might take a little while (day? more?) to update. But at the moment it's us who is out of date, not the CDN...

On the other hand, this wouldn't work to make it part of the release script.

I'm wonder about switching from Bower to NPM though.  At least that I understand, and `npm publish` *is* part of the release script.

**Comment by edemaine** [comment]
Never mind, I got bower working: `rm -rf bower_components/katex` followed by `bower install katex` fixed it.

**Comment by ylemkimon** [comment]
It's little out of scope, but I think Github pages should be a separate repo, i.e., katex.github.io on KaTeX organization, as main repo Git history can become large.

**Comment by ylemkimon** [comment]
Also, I wonder if there are interests or needs for online demo(sandbox) for each build(PR and push), like Babel REPL, using build artifacts.

**Comment by edemaine** [comment]
@ylemkimon These both seem like good ideas.  It'd already be cool to compare outputs across released versions.  PR versions would be even neater -- handy for testing.  Can you link to a similar demo for another project?

@kevinbarabash What do you/KA think of moving the gh-pages branch into a new repo in the KaTeX project?

**Comment by ylemkimon** [comment]
@edemaine AFAIK, Babel as mentioned above: https://github.com/babel/babel/pull/7450#issuecomment-369175564.

**Comment by ylemkimon** [comment]
I think at least we need to migrate to NPM, as `bower` is deprecated.

**Comment by kevinbarabash** [comment]
@edemaine I'm interested in moving in this direction.  There are some issues that I need to work through with the organization about making this change.  I've been busy with other stuff lately and so have put it on the back burner.  I'll pick this up again in April when I have some more time.

**Comment by edemaine** [comment]
Fixed by #1180
